sagemaker-spark-sdk/src/test/scala/com/amazonaws/services/sagemaker/sparksdk/transformation/deserializers/ProtobufResponseRowDeserializerTests.scala

```diff
@@ -179,4 +179,29 @@ class ProtobufResponseRowDeserializerTests extends FlatSpec with Matchers with M
     assert(schemaFieldNames(0) == protobufKeys(0))
     assert(schemaFieldNames(1) == protobufKeys(1))
   }
+
+  it should "create a factorization machines regressor deserializer with " +
+    "the correct schema and fields" in {
+    val regressorDeserializer = new FactorizationMachinesRegressorProtobufResponseRowDeserializer()
+    val schemaFieldNames = regressorDeserializer.schema.fieldNames
+    assert(schemaFieldNames.contains("score"))
+    val protobufKeys = regressorDeserializer.protobufKeys.get
+    assert(protobufKeys.contains("score"))
+    assert(schemaFieldNames(0) == protobufKeys(0))
+  }
+
+  it should "create a factorization machines  binary classifier deserializer with " +
```

Comment:
Could you remove the extra space between "machines" and "binary"?

Reply:
will remove